types/reactjs-localstorage/index.d.ts

```diff
@@ -0,0 +1,14 @@
+// Type definitions for reactjs-localstorage 1.0
+// Project: https://github.com/null-none/react-localstorage
+// Definitions by: Rick Wilson <https://github.com/rwilson504>
+//                 Dan Cox <https://github.com/powerappsdev>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
```

Comment:
Please add a minimally supported typescript version

Reply:
Will do

Reply:
The guidance may have been updated since last time I checked, but this is not generally necessary if the tests are passing.